CollAction/Views/Emails/DonationThankYou.cshtml

```diff
@@ -0,0 +1,17 @@
+﻿@model CollAction.Models.EmailViewModels.DonationThankYouEmailViewModel
+Hi @Model.Name!,<br>
+<br>
```

Comment:
Why no paragraphs here? Eg. <p>

Reply:
Done.

Mostly needed the paragraph tags because you can't use raw text inside if-statements. They're better too semantically, so I changed the donation e-mail as well.